phasespace/fromdecay/genmultidecay.py

```diff
@@ -0,0 +1,291 @@
+import itertools
+from typing import Callable, Dict, List, Set, Tuple, Union
+
+import tensorflow as tf
+import tensorflow.experimental.numpy as tnp
+from particle import Particle
+
+from phasespace import GenParticle
+
+from .mass_functions import DEFAULT_CONVERTER
+
+MASS_WIDTH_TOLERANCE = 0.01
+DEFAULT_MASS_FUNC = "relbw"

```

Comment:
What about making the class variables? So they could be changed by the user easily

Reply:
Changed in decf916